docs/guides/custom-server.md

```diff
@@ -29,6 +30,33 @@ available in KeystoneJS include:
   The following are some possible ways of setting up a custom server, roughly in
   order of complexity.
 
+## You might not need a custom server if...
+
+If all you want to do is some basic configuration of the default Express instance, you don't need a
+custom server! The KeystoneJS CLI accepts an additional `configureExpress` export in your `index.js` file,
+like so:
+
+```javascript
+module.exports = {
+  configureExpress: app => {
+    /* ... */
+  },
+};
+```
+
+This function takes a single `app` parameter. The running Express instance will be passed to this function
+before any middlewares are set up, so you can perform any Express configuration you need here. For example:
+
+```javascript
+module.exports = {
+  configureExpress: app => {
+    app.set('view engine', 'pug')
+  },
+};
+```
+
+This export can also be used with a custom server (see below).
```

Comment:
```suggestion
```

Reply:
Remove the reference below as you probably would not use `configureExpress` with a custom server. My assumption is this was an attempt to document it quickly but the detail above now covers this.

Reply:
Fixed, and also removed the examples in the custom server code.